docs/make.jl

```diff
@@ -3,6 +3,11 @@ using Literate
 using JuMP
 using Test
 
+# Pass --fast as an argument to skip rebuilding the examples and running
+# doctests. Only use this argument to rapidly test small changes to the
+# Markdown. _Never_ set it in production.
+const _FAST = get(ARGS, 1, "") == "--fast"
```

Comment:
I can remove or split this into a separate PR if needed, but it was useful when compiling the list of missing docstrings, not to have to repeatedly generate the literate files.

Reply:
Why not `findfirst(isequal("--fast"), ARGS) !== nothing` ?